fixtures/lightmaxx/pure-ligt.json

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "pure ligt",
+  "shortName": "muvy spotone",
+  "categories": ["Moving Head", "Color Changer", "Dimmer", "Effect", "Strobe"],
+  "meta": {
+    "authors": ["theo"],
+    "createDate": "2023-12-02",
+    "lastModifyDate": "2023-12-02"
+  },
+  "links": {
+    "manual": [
+      "https://www.manualslib.com/download/1916522/Purelight-Muvy-Spotone.html"
```

Comment:
I was unable to pass the CAPTCHA here so I can't verify this but here's another URL that doesn't require one: https://manualspro.net/287651-purelight-spotone-led-moving-head-spot-user-manual